rotkehlchen/db/dbhandler.py

```diff
@@ -2768,27 +2768,49 @@ def get_latest_asset_value_distribution(self) -> List[DBAssetBalance]:
             ignored_assets = self.get_ignored_assets(write_cursor)
 
         with self.conn.read_ctx() as cursor:
+            treat_eth2_as_eth = self.get_settings(cursor).treat_eth2_as_eth
             cursor.execute(
                 'SELECT time, currency, amount, usd_value, category FROM timed_balances WHERE '
                 'time=(SELECT MAX(time) from timed_balances) AND category = ? ORDER BY '
                 'CAST(usd_value AS REAL) DESC;',
                 (BalanceType.ASSET.serialize_for_db(),),  # pylint: disable=no-member
             )
             asset_balances = []
+            eth_balance: Dict[str, str] = {
+                'time': '0',
```

Comment:
Is there any reason to treat this as string instead of integer? In the database and everywhere else we use integers for timestamps I believe